javascript/javascript3/week1/owncooljson.js

```diff
@@ -0,0 +1,36 @@
+//#1 Create your own json file:
+const myIdentity = {
+  Ethnicity: "African",
+  Nationality: "Southern Cameroonian",
+  Tribe: "Ejagham",
+  Sex: "Male",
+  Age: 60,
+  Name: { Firstname: "John", Middlename: "Ako Ayukngat", Lastname: "Ojong" },
+  Hobby: ["Grilling", "Beef Butchery", "Reading", "Fitness"],
+};
+console.log(JSON.stringify(myIdentity));
+
+const myIdentityParse = JSON.stringify(myIdentity);
+console.log(JSON.parse(myIdentityParse));
+
+//#2 Find a cool api:
+
+fetch("https://www.breakingbadapi.com/api/")
+  .then((response) => response.json())
+  .then((data) => {
+    console.log(data);
+
+    const getDeathApi = data.deaths;
+    console.log(getDeathApi);
+  });
+
+fetch("https://breakingbadapi.com/api/deaths")
+  .then((response) => response.json())
+  .then((deathInstances) => {
+    console.log(deathInstances);
```

Comment:
same thing goes for console.logs, It's better you wrap this in a function, and return the number of deaths, and then the console.log shows the number in the console. Don't get me wrong, this works, just suggesting a better practice.